packages/icons/icons.json

```
@@ -1745,217 +1745,217 @@
         "group": "table",
         "codepoint": 61791
     },
     {
         "displayName": "Arrow: top left",
         "iconName": "arrow-top-left",
-        "tags": "direction, north west",
+        "tags": "direction, north west, arrow, up left arrow, top left arrow, cursor, up left, arrow up left",
         "group": "interface",
         "codepoint": 61735
     },
     {
         "displayName": "Arrow: top right",
         "iconName": "arrow-top-right",
-        "tags": "direction, north east",
+        "tags": "direction, north east, top right arrow, upper right arrow, arrow, top right, pointer, up, go, arrow diagonal, open in new tab, arrow up right, upper right corner",
         "group": "interface",
         "codepoint": 61736
     },
     {
         "displayName": "Arrow: bottom right",
         "iconName": "arrow-bottom-right",
-        "tags": "direction, south east",
+        "tags": "direction, south east, arrow down right, down left arrow, diagonal arrow, diagonal down left arrow",
         "group": "interface",
         "codepoint": 61731
     },
     {
         "displayName": "Arrow: bottom left",
         "iconName": "arrow-bottom-left",
-        "tags": "direction, south west",
+        "tags": "direction, south west, arrow, bottom left direction, point, minimize, collapse, go to, here, this, down and to the left, right down arrow, diagonal",
         "group": "interface",
         "codepoint": 61730
     },
     {
         "displayName": "Mugshot",
         "iconName": "mugshot",
-        "tags": "person, photograph, picture,",
+        "tags": "person, photograph, picture, portrait, profile picture, mark zuckerberg, user",
         "group": "interface",
         "codepoint": 62029
     },
     {
         "displayName": "Headset",
         "iconName": "headset",
-        "tags": "headphones, call, communication",
+        "tags": "headphones, call, communication, audio",
         "group": "media",
         "codepoint": 61926
     },
     {
         "displayName": "Text highlight",
         "iconName": "text-highlight",
-        "tags": "selector, content",
+        "tags": "selector, content, type, input, text insert, select, highlight, text input, text box, input text, search, enter text, text area",
         "group": "editor",
         "codepoint": 62181
     },
     {
         "displayName": "Hand",
         "iconName": "hand",
-        "tags": "gesture, fingers",
+        "tags": "gesture, fingers, raise hand",
         "group": "interface",
         "codepoint": 61920
     },
     {
         "displayName": "Chevron: backward",
         "iconName": "chevron-backward",
-        "tags": "skip, direction",
+        "tags": "skip, direction, k",
         "group": "interface",
         "codepoint": 61769
     },
     {
         "displayName": "Chevron: forward",
         "iconName": "chevron-forward",
-        "tags": "skip, direction",
+        "tags": "skip, direction, skip to end, strikeout",
```

```suggestion
        "tags": "skip, direction, end",
```

what does "strikeout" mean in this context?